releasenotes/notes/protein-folding-result-b344ac3c7f48e3ca.yaml

```diff
@@ -0,0 +1,20 @@
+---
+features:
+  - |
+    Completed :class:`~qiskit_nature.results.protein_folding_result.ProteinFoldingResult` .
+    The result of :class:`~qiskit_nature.problems.sampling.protein_folding.protein_folding_problem.ProteinFoldingProblem`
+    is interpreted by this class and translates it into relevant information such as the turns defining the main and side chains
+    of the protein, the positions of each aminoacid, as well as a plot of the molecular structure. It does so by using 3 new classes.
+    Added :class:`~qiskit_nature.results.protein_folding_result.protein_folding_tools.protein_shape_decoder.ProteinShapeDecoder`. This class takes a raw bitstring result
+    and decodes it to yield the turns defining the shape of the molecule.
+    Added :class:`~qiskit_nature.results.protein_folding_result.protein_folding_tools.protein_shape_generator.ProteinShapeFileGen`. This class converts the shape of the molecule
+    into Cartesian coordinates for each aminoacid in the chain.
+    Added :class:`~qiskit_nature.results.protein_folding_result.protein_folding_tools.protein_plotter.ProteinPlotter`. This class creates a plot of the molecule.
+
+    .. code-block:: python
+
+        result = protein_folding_problem.interpret(raw_result)
+        #Get xyz file
+        result.protein_xyz.save_xyz_file()
+        #Generate plot
+        figure = result.plot_folded_protein()
```

Comment:
Is this code correct? I would have thought its "Save protein as xyz file" as comment and `result.save._xyz_file()`.